### ISSUE TITLE
CI for s390x

### ISSUE THREAD

**Issue by AlekseiNikiforovIBM** [pytorch/pytorch#99305]
It'd be great to have s390x CI for pytorch. It'd help to improve support for s390x in pytorch.

It's possible to request s390x machine for CI for pytorch. Only question is who'll manage it: pytorch maintainers or us. For me first option would be more convenient, but second one is also possible.

**Comment by malfet** [comment]
Please correct me if I'm wrong, but [S/390](https://en.wikipedia.org/wiki/IBM_System/390) has been discontinued 22 years ago,  wasn't it? In that case, how long would it take to build PyTorch on the system?

**Comment by AlekseiNikiforovIBM** [comment]
Under s390x usually [this is meant](https://en.wikipedia.org/wiki/IBM_Z)

**Comment by malfet** [comment]
Thank you for the clarification. Is there a way to indeed get a remote access to the system, to see how easy/hard would it be to setup a CI there

**Comment by AlekseiNikiforovIBM** [comment]
@malfet, a cloud instance should be available for creation. Which email address can I use to invite you there? meta, fb, gmail or some other?

**Comment by v1gnesh** [comment]
I think this may help - [GitHub Actions runner for IBM Z and LinuxONE!](https://community.ibm.com/community/user/ibmz-and-linuxone/blogs/mingmay-pang2/2024/02/05/github-actions-runner-for-ibm-z-and-linuxone)

**Comment by kiszk** [comment]
@v1gnesh Yes, thank you. I have just sent a request to allocate an instance from https://www.ibm.com/community/z/open-source/virtual-machines-request/

**Comment by AlekseiNikiforovIBM** [comment]
Closing as implemented
